### ISSUE TITLE
Add ability to autoplay gifs in page search results

### ISSUE THREAD

**Issue by lollyhunter** [hydrusnetwork/hydrus#825]
Currently you have to click on a gif for it to start playing in the media viewer. Adding the ability to autoplay multiple gifs displayed on the panel on the right instead of just showing the thumbnails would be nice.

**Comment by Suika** [comment]
Currently MPV is being used to display the "preview" window to the side and the actual file once you open it.
Playing them, would mean that for every gif in your view, a MPV instance would be created and then killed once it leaves the view window.

Also would mean that there would be a lot of access to disk, to display all the files.

**Comment by lollyhunter** [comment]
yeah you would probably want to paginate it, and maybe make each gif medai viewer bigger, so there were less on the screen at a time. I'm thinking of something similar to the gif page of sex.com.
